CoreConversationView/src/au/gov/asd/tac/constellation/views/conversationview/TranslationConversationContributionProvider.java

```diff
@@ -102,36 +102,6 @@ public TranslationContribution(final String graphId, final ConversationMessage m
                         editTranslationTextArea.deleteText(selection);
                     }
```

Comment:
All of setOnKeyPressed block can be removed?

Reply:
done